main.py

```diff
@@ -0,0 +1,47 @@
+"""
+Python Based Movie Booking Management System Using Filesystem and Json
+Drive Code Main
+"""
+
+import time
+
+from theater import Theater
+from utils import (
+    custom_print_msg,
+    user_input,
+    user_input_int_type,
+    create_required_json_files,
+    create_user_transactions_json_file,
+)
+
+
+def main() -> None:
+    """
+    main Driver function for Step by Step Process
+    @return:None
```

Comment:
if you are defining output type `-> None` then no need to define `@return:None`, same for parameters